src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Kiezkassen.ts

```diff
@@ -24,7 +25,9 @@ export var detailDirective = (adhConfig : AdhConfig.IService, adhHttp : AdhHttp.
                 heading: "Tab2",
                 content: "foo2"
             }];
-
+            scope.$watch(() => column.$scope.shared.isShowMap, function(value) {
```

Comment:
I think it would be better to use `column.$scope.$watch("shared.isShowMap", ...)`. Also this should be lambda syntax instead of function syntax.